pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <artifactId>unlogged-sdk</artifactId>
     <groupId>video.bug</groupId>
-    <version>0.6.3</version>
+    <version>0.6.102</version>
```

Comment:
remove this, you can keep this change local to yourself

Reply:
okay using version `0.6.3`.